src/services/response-service.ts

```diff
@@ -55,6 +56,13 @@ export default class ResponseService extends PercyClientService {
       return
     }
 
+    const responseBodySize = (await response.buffer()).length
+    if (responseBodySize > Constants.MAX_FILE_SIZE_BYTES) {
```

Comment:
Per discussion, let's reorganize this to `stat` the local file copy afterward, instead of reading the buffer and checking it here before writing the local copy (ie. optimize for the 99% case of not skipping, and use stat because it's fast and avoids the double buffer read)

Reply:
Fixed with https://github.com/percy/percy-agent/pull/243/commits/941db82a027f68cf0078319e211386e4d9f34936